src/alert-manager/src/cluster-utilization/send_alert.py

```diff
@@ -49,15 +49,39 @@ def datetime_to_hours(dt):
     return dt.days * 24 + dt.seconds / 3600
 
 
+def check_timestamp_within_7d(timestamp):
+    """
+    check if a timestamp is within 7 days
+    """
+    return datetime.fromtimestamp(int(timestamp/1000), timezone.utc) > datetime.now(timezone.utc) - timedelta(days=7)
+
+
+def get_jobs_in_7d(rest_url):
```

Comment:
Please change to another name. This function name is not correct, since it may return job before 7 days and contain dup job.

Reply:
> Please change to another name. This function name is not correct, since it may return job before 7 days and contain dup job.

Fixed, thx